Gemfile

```diff
@@ -4,31 +4,34 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.22'
+gem 'rails', '4.1.5'
 gem 'rack-cache', '< 1.3.0'
```

Comment:
This gem should be removed, it was a pin for a dep in actionpack 3.x, no longer there.